### DISCUSSION TITLE
Changing Encoded Symmetry in ORTool Examples

### DISCUSSION THREAD

**Discussion by Savillian** [ORTools4MTEX/ORTools#65] [Q&A] [answered]
Hi everyone!

I wanted to ask if anyone's had success changing the space groups listed for ORTool Examples (e.g., Example 2 with Beta-Ti reconstructions). MTEX defaults to using incorrect space groups (432 for Beta-Ti and 622 for Alpha-Ti, when they should be m-3m and 6/mmm respectively). I wanted to change these values in developing a new reconstruction workflow.

I'd assumed using a standard redefinition such as ebsd.CSList{1,3}.pointGroup = '6/mmm' would work, but this is incompatible with the way the ebsd variable is structured. I get the following error: 

```
ebsd.CSList{1,3}.pointGroup  = '6/mmm';
In class 'symmetry', no set method is defined for dependent property 'pointGroup'.  A dependent property needs a set method to
assign its value.

Error in indexing (line 95)
      ebsd = builtin('subsasgn',ebsd,s,b);
```
I imagine the way variables are configured in MATLAB vs MTEX vs ORTools is likely the culprit and I'm not catching something obvious. Any help's appreciated.

Thanks in advance!
Alec

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @Savillian,

The example scripts within ORTools use the default set of MTEX maps for educational and illustrative purposes only.
The choice of these maps is because they are packaged within MTEX and so, are readily available to everyone.
The idea is that users learn how to use MTEX and ORTools using these example maps and then swap over to their own maps when they are comfortable with the scripts.

That said, the default maps are also routinely used by us developers to test algorithms.
The different types of maps are acquired from a range of sources and enable us to create scripts that work seamlessly across OEM systems and platform types.

If you wish to use change the crystal symmetries of any ebsd map, I'd advise you first to be careful with data modifications and then suggest the following route:

```matlab
clc; close all; close hidden

%% Load the mtex map data
mtexdata alphaBetaTitanium


%% Define a new ebsd map
% Define new crystal symmetries (in the same order as the original map)
CS_bcc = crystalSymmetry('m-3m', [1.5708 1.5708 1.5708], 'mineral', 'beta');
CS_hcp = crystalSymmetry('6/mmm', [1.5708 1.5708 2.0944], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'alpha');
CSList = {'notIndexed',CS_bcc,CS_hcp};

% Create a new ebsd variable
newebsd = EBSD(rotation.byEuler(ebsd.rotations.phi1(:),ebsd.rotations.Phi(:),ebsd.rotations.phi2(:)),...
    ebsd.phaseId,...
    CSList,...
    ebsd.prop);

% Save the new ebsd variable as a map
pfName = [pwd,'\alphaBetaTitanium_new.ctf'];
export_ctf(newebsd, pfName);
```

<ins>**Lastly, and importantly, if an explanation from the maintainers resolves your issue, please press the "Mark as answer" button on the appropriate response. Your assistance in this regard enables us to close discussions as complete. **</ins>

Hope this helps.

Warm regards,
Azdi

**Reply by Savillian** [reply]
Hi Azdi,

Thanks for the quick reply! I didn't mean to insinuate anything in ORTools is wrong (far from it, the capabilities are amazing!) - apologies if that's how my comment on point groups came across.

I hadn't thought about using the standard MTEX crystal symmetry designation process and then redefining the map. Great workaround, thanks! 

All the best,
Alec

**Reply by AzdiarGazder** [reply]
Hi @Savillian,

You're very welcome to the workaround.
I am very happy to learn you're enjoying the capabilities of ORTools.
I hope the toolbox will be of use to your research, and look forward to reading about your exciting finds using these scripts.

And please don't worry - I just thought I'd clarify the reason why the default set of MTEX maps was used in ORTools. :)

Warm regards,
Azdi
